src/SmartTransactionsController.ts

```diff
@@ -126,13 +133,28 @@ export default class SmartTransactionsController extends BaseController<
           tradeTxFees: undefined,
         },
         liveness: true,
+        livenessByChainId: {
+          [CHAIN_IDS.ETHEREUM]: true,
+          [CHAIN_IDS.GOERLI]: true,
+        },
+        feesByChainId: {
+          [CHAIN_IDS.ETHEREUM]: {
+            approvalTxFees: undefined,
+            tradeTxFees: undefined,
+          },
+          [CHAIN_IDS.GOERLI]: {
+            approvalTxFees: undefined,
+            tradeTxFees: undefined,
+          },
+        },
       },
     };
-
+    this.setIntervalLength(this.config.interval || DEFAULT_INTERVAL);
```

Comment:
this can/should be removed but I need to move the `this.initialize()` call above. Without moving this we were always using the DEFAULT_INTERVAL. done here: https://github.com/MetaMask/smart-transactions-controller/pull/237/commits/8e2ee0de68987a45c13375a2770300251bf08e7f